snuba/stateful_consumer/consumer_context.py

```diff
@@ -55,7 +55,9 @@ def _build_state(
         state_class: StateType,
     ) -> State[ConsumerStateCompletionEvent, ConsumerStateData]:
         if state_class == ConsumingState:
-            return ConsumingState(self.__main_consumer)
+            return ConsumingState(self.__consumer_builder)
+        elif state_class == CatchingUpState:
+            return ConsumingState(self.__consumer_builder)
```

Comment:
Is this right? Shouldn't this be `CatchingUpState`?

Reply:
This is a merge mistake.